eng/versioning/version_client.txt

```diff
@@ -490,13 +490,15 @@ io.clientcore:http-stress;1.0.0-beta.1;1.0.0-beta.1
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
-unreleased_com.azure:azure-messaging-servicebus;7.18.0-beta.2
-unreleased_com.azure:azure-messaging-eventhubs;5.19.0-beta.3
-unreleased_com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.20.0-beta.3
+
+unreleased_com.azure:azure-messaging-servicebus;7.18.0
```

Comment:
I thought the Service Bus version being released was 7.17.4?

Reply:
Urgh, Thanks good catch